core/services/keystore/keys/ocr2key/solana_keyring.go

```diff
@@ -48,15 +50,22 @@ func (skr *solanaKeyring) Sign(reportCtx ocrtypes.ReportContext, report ocrtypes
 }
 
 func (skr *solanaKeyring) Sign3(digest types.ConfigDigest, seqNr uint64, r ocrtypes.Report) (signature []byte, err error) {
```

Comment:
What would be nice as a follow up is to have tests that sign reports w/ these funcs and verify the sigs using the contract code (similar to the current ContractTransmitter test for EVM)